### DISCUSSION TITLE
Why don't you up plugin to https://wordpress.org/plugins?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [wp-graphql/wp-graphql#1491] [Q&A] [answered]
I wonder why don't you up plugin to WP? It more easy to search and setup..

**Comment by moonmeister** [comment]
I believe this will happen with 1.0.

**Comment by Soean** [comment, marked as answer]
It's on wp.org now: https://wordpress.org/plugins/wp-graphql/
